qabelbox/src/main/java/de/qabel/desktop/repository/sqlite/SqliteContactRepository.java

```diff
@@ -195,4 +204,71 @@ public synchronized Contact findByKeyId(Identity identity, String keyId) throws
             throw new EntityNotFoundExcepion("exception while searching contact: " + e.getMessage(), e);
         }
     }
+
+    @Override
+    public Collection<Pair<Contact, List<Identity>>> find(Identities identities, String searchString) throws PersistenceException {
+        try {
```

Comment:
Is there really a need to wrap the try arround the complete method and not just the SQL stuff?